packages/ui/src/components/organisms/DynamicForm/components/RSJVInputAdaters/TextInputAdapter/helpers/detectFieldType.ts

```diff
@@ -2,7 +2,7 @@ import { RJSFInputProps } from '@/components/organisms/DynamicForm/components/RS
 import { TextInputFieldType } from '../types';
 
 export const detectFieldType = (fieldProps: RJSFInputProps<unknown>): TextInputFieldType => {
-  if (fieldProps.schema.oneOf) return 'select';
+  if (fieldProps.schema.oneOf || fieldProps.schema.enum) return 'select';
```

Comment:
Do we need anyOf?

Reply:
We don't use it at the moment. I prefer not to implement it as I won't test it against a real use case.